pkg/server/container_create.go

```diff
@@ -225,6 +226,18 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	if username := securityContext.GetRunAsUsername(); username != "" {
 		specOpts = append(specOpts, oci.WithUsername(username))
 	}
+	if gid := securityContext.GetRunAsGroup(); gid != nil {
+		if securityContext.GetRunAsUser() == nil || securityContext.GetRunAsUsername() == "" {
```

Comment:
Above comment block should be modified to mention groupid

Reply:
uid and username variables should be promoted in scope so they can be used here in this if nil check

Reply:
Will do.